app/actions/views/select_server.js

```diff
@@ -12,6 +13,24 @@ export function handleServerUrlChanged(serverUrl) {
     };
 }
 
+export function handleLoginOptions() {
+    return async (dispatch, getState) => {
+        const {config, license} = getState().entities.general;
+
+        let options = 0;
+        if (config.EnableSaml === 'true' && license.IsLicensed === 'true' && license.SAML === 'true') {
+            options++;
```

Comment:
it would be clearer but then we want to have it the other way for when we add other sign in methods